docs/SUMMARY.md

```diff
@@ -29,7 +29,7 @@
   * [9. Styles](learning-frontity/styles.md)
   * [10. Code Splitting](learning-frontity/code-splitting.md)
   * [11. Head](learning-frontity/head.md)
-* [🏗 Architecture](architecture.md)
+* [🏗 Architecture](architecture/architecture.md)
```

Comment:
Don't you have to change this link to `architecture/README.md`?

Reply:
Already done